project/ProjectPlugin.scala

```diff
@@ -56,6 +56,8 @@ object ProjectPlugin extends AutoPlugin {
       orgLicenseSetting := MITLicense,
       headerLicense := Some(
         HeaderLicense.MIT(replaceableYear(startYear.value), "Beyond the lines")),
+      // disable license headers on source files because it's too complicated, owing to us forking the project
+      excludeFilter.in(unmanagedSources.in(headerCreate)) := HiddenFileFilter || "*.*",
```

Comment:
Whoops, this also excluded all files from being compiled! The Travis build passed by doing no compilation and then running no tests.

Looking into a fix now.